SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialActorChannel.h

```diff
@@ -136,6 +137,12 @@ class SPATIALGDK_API USpatialActorChannel : public UActorChannel
 
 		if (EntityId != SpatialConstants::INVALID_ENTITY_ID)
 		{
+			// Make sure we have spatial authority before we replicate with Offloading, as we pretend to have local authority
```

Comment:
```suggestion
			// If the entity already exists, make sure we have spatial authority before we replicate with Offloading, because we pretend to have local authority
```